### PR TITLE
#1469 Add sorting function in table component

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1469 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Mui/Table/SortTable.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    if (b[orderBy] < a[orderBy]) {
+      return -1;
+    }
+    if (b[orderBy] > a[orderBy]) {
+      return 1;
+    }
+    return 0;
+  };
+
+  const stableSort = (array, cmp) => {
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Table/SortTable.js]: component

File: ohara-manager/client/src/components/common/Mui/Table/SortTable.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+  if (isLoading) return <TableLoader />;
+
+  return (
+    <Paper>
+      <Table>
+        <TableHead>
+          <TableRow>
+            {headRows.map(row => {
+              const { id, label, sort = true } = row;
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Table/SortTable.js]: sortable

**Comment by konekoya** [comment]
retry manager-it

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by chia7712** [comment]
@wu87988622 our jenkins is slow now. And I am digging in it now. Hence please retry you PR tomorrow.

**Comment by wu87988622** [comment]
> @wu87988622 our jenkins is slow now. And I am digging in it now. Hence please retry you PR tomorrow.


ok i know, thank you
You can close my task.

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager-it

**Pull request merged**